src/rust/ensogl/src/system/gpu/data/texture/class.rs

```diff
@@ -161,16 +207,14 @@ where S:StorageRelation<I,T> {
         let storage    = storage.into();
         let context    = context.clone();
         let gl_texture = context.create_texture().unwrap();
-        Self {storage,gl_texture,context}
+        let parameters = Parameters::default();
+        Self {storage,gl_texture,context,parameters}
     }
 
-    /// Sets the texture wrapping parameters.
-    pub fn set_texture_parameters(context:&Context) {
-        let target = Context::TEXTURE_2D;
-        let wrap   = Context::CLAMP_TO_EDGE as i32;
-        context.tex_parameteri(target,Context::TEXTURE_MIN_FILTER,Context::LINEAR as i32);
-        context.tex_parameteri(target,Context::TEXTURE_WRAP_S,wrap);
-        context.tex_parameteri(target,Context::TEXTURE_WRAP_T,wrap);
+
+    /// Sets the texture wrapping parameters without requiring self.
```

Comment:
I dont understand the comment. What does "without requiring self" mean?

Reply:
This doc should no longer be there. Removed.